tests/api-watch-diff/test.ml

```diff
@@ -166,10 +143,23 @@ let modify_type_in_value_signature =
     |}
 
 let%expect_test "Modifying a type used in a value" =
-  let result =
-    Api_watch_diff.diff_interface ~reference:ref_signature
-      ~current:modify_type_in_value_signature
+  let reference =
+    Test_helpers.compile_interface
+      {|
+    type t = int
+    type unused_type = string
```

Comment:
we can remove this line from both signatures as it is now irrelevant to this test!